src/index.json

```diff
@@ -46,6 +46,54 @@
                 "sha256Digest": "4ac7b8a4a89eda68d9d1a07cc5edd9b1a2b88421e2aa9a9e5b86a241f127775f"
             }
         ],
+        "ai-examples": [
+            {
+                "downloadUrl": "https://aladdinprodwestus.blob.core.windows.net/aladdin-extensions/ai_examples-0.1.0-py2.py3-none-any.whl",
```

Comment:
download link should be generated by CLI team, and you don't need to provide it.

Reply:
OK, I was unclear how that was supposed to happen. I'll remove it for now.

Reply:
Err, how am I supposed to update the `index.json` then? I had been using the `azdev extension publish` command to do that up till now, but if y'all will be publishing it, that doesn't make sense to use. The `azdev extension update-index` command looks like the right thing, but it also needs a URL to work.

Reply:
@Juliehzl any update on this?

Reply:
@mirdaki, currently the workflow is:
- You build a whl file
- You send whl file to us
- We'll upload your whl file to our storage (as you don't have permission to upload to our storage)
- We tell you the downloadUrl
- You update the index.json with the downloadUrl we provide to you

In the future (one or two sprints later),
All these manual steps will be automated in our extension release pipeline.